translations/de/hints/bryukh.html

```diff
@@ -0,0 +1,40 @@
+<body data-author="bryukh">
+<div class="question-answer">
+    <div class="question">
+        Ich habe keine Ahnung wo ich mit der L&ouml;sung anfangen soll.
```

Comment:
If you leave out the word "Lösung" the sentence would become:
_Ich habe keine Ahnung **wie** ich anfangen soll._ - this really sound weird, weirder then:
_Ich habe keine Ahnung **wo** ich anfangen soll._